tox.ini

```diff
@@ -3,12 +3,13 @@ envlist = coverage-clean, py27, py33, py34, py35, coverage-report
 
 [testenv]
 commands =
-    coverage run setup.py ftest
+    coverage run --source=zope.app.publication -m zope.testrunner --test-path=src --auto-color --auto-progress
```

Comment:
`--source=zope.app.publication` should not be required, since we specify `source = zope.app.publication` in `./coveragerc`.